.github/workflows/docs.yml

```diff
@@ -12,12 +12,16 @@ jobs:
     runs-on: ubuntu-latest
     name: Docs
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js 12.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    - run: npm ci
-      name: Install dependencies
-    - run: npm run lint:docs
-      name: Lint
\ No newline at end of file
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+            fetch-depth: 0
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          cache: 'npm'
+      - run: npm ci
```

Comment:
I'm still confused of when you need to say `shell: bash`. Should we add it here?

Reply:
No, it doesn't need to be added here as this is a workflow definition (i.e it is under `.github/workflows`)